src/main/java/org/openhds/errors/model/ErrorLog.java

```diff
@@ -19,6 +21,7 @@
     @Column(length=65535)
     private String dataPayload;
 
+    @NotEmpty
```

Comment:
I think all our validation annotations should contain messages.  This will help debug.